docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1233,7 +1233,8 @@ See the :ref:`hive_examples` for more information.
     -
   * - ``csv_separator``
     - The CSV separator character. Requires CSV format.
-    -
+    - Defaults to ``,``. You can use other separators such as ``|`` or use
```

Comment:
Why is there another - at the beginning of this line .. I dont think this is correct or even builds..